test/Factory/white-list.yml

```diff
@@ -0,0 +1,204 @@
+# 
+# This file describes a white list of files in /usr/etc found in
+# http://download.suse.de/ibs/SUSE:/Factory:/Head/standard/repodata/
+# which are not touched by YAST or which are already handled by YAST correctly.
+#
+#  files:
+#    < files/dirs in /usr/etc >
+#  defined_by:
+#    < none YAST packages which contain/touch these files >
+#   yast_support:
+#    < YAST packages which handle these files ( or "not used by YAST" ) >
+#
+
+- files:
+    - /usr/etc/default
+    - /usr/etc/default/runuser
+    - /usr/etc/default/su
+  defined_by:
+    - util-linux
+  yast_support:
+    - "not used by YAST"
```

Comment:
This "not used by YAST" magic "constant" looks strange, you can easily make a typo there. If it is not used by YaST then I'd either use `nil` or (IMHO better) remove the `yast_support` key completely. Missing `yast_support` => no YaST support.